src/macaron/slsa_analyzer/analyzer.py

```diff
@@ -513,58 +603,144 @@ def add_component(
 
         Raises
         ------
-        RepoNotFoundError
-            No corresponding repository was found.
         PURLNotFoundError
             No PURL is found for the component.
         DuplicateCmpError
             The component is already analyzed in the same session.
         """
-        # Note: the component created in this function will be added to the database.
```

Comment:
Please add this comment back.

Reply:
Fixed in https://github.com/oracle/macaron/pull/401/commits/f66b0be2bf3c2556eed5885315d39cd66b792306